### PR TITLE
created some reusable rule sets

### PR DESCRIPTION
**Pull request by markkramerus**
I have to explain an obsession of mine. If you look at the JSON of a StructureDefinition (e.g., http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/StructureDefinition-mcode-patient-bundle.profile.json.html), you will find a whopping huge file.  What makes it so big? Well, if you open it up and search for the words "no stigma" you will find multiple copies of the same text block, over and over. In the patient bundle, the same text block is repeated 202 times. There are several different, quite large text blocks associated with elements that have the same definition and comment (mostly because almost all elements have extensions and some have modifier extensions).

I wrote some rule sets that clean out these useless text blocks. It turns out you can't null them out, but you can replace them with single-character strings.

You can call me obsessive about this if you want. Go ahead.

### CODE REVIEW

**Comment by markkramerus** [comment]
Another thing to look at, especially for Max, is the handling of Tumor identifiers. I do not think the system you created would work as it was (or is). You sliced the TumorSpecimen array on the type of identifier, which is reasonable only if such an identifier were unique to tumor identification. There was not corresponding constraint place on the identifiers of Tumors, however. I tried to create a requirement for a particular type of identifier in Tumor.identifier, and link that through that type to TumorSpecimen. I tried to use an existing code for the identifier.type, as you will see.

File: input/pagecontent/conformance-profiles.md [hunk 1]
```diff
@@ -82,42 +82,45 @@ An mCODE data element is **required** if any of the following criteria are met:
...
+{: .grid }
 
-##### Viewing Must Support Flags
-To see which elements have <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags, consult the "Snapshot Table" view of the profile. The "Differential Table" view hides <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags inherited from the parent profile. The "Snapshot Table (Must Support)" view reflects the IG Publisher's interpretation of how <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> flags translate to support requirements, which may or may not coincide with the US Core/mCODE interpretation.
+#### Non-Must Support Elements
 
-#### Interpretation of <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" >S</span> Flags
+Data elements in mCODE that *do not have* MS obligations MAY be supported. If a non-obligatory element is supported, the implementation of that element MUST conform to the corresponding profile. Moreover, any data element that would reasonably be expected to conform to that mCODE data element SHOULD conform to that element.
```
Comment (mlterryMitre) [inline input/pagecontent/conformance-profiles.md:117]: This is confusing. Needs examples to clarify.

**Review by mlterryMitre — CHANGES_REQUESTED** [review]
The conformance comment on non-supported must-support elements needs at least one example to clarify it.

File: input/pagecontent/conformance-profiles.md (hunk 1 above)
Comment (markkramerus) [reply input/pagecontent/conformance-profiles.md:117]: I added an example and simplified the language.

**Pull request merged**